data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -64,6 +65,27 @@ def _climate_factor(self) -> bool:
         # Low income: In 60th percentile or above for percent of block group population
         # of households where household income is less than or equal to twice the federal
         # poverty level. Source: Census's American Community Survey]
+
+        # I am thinking that we make constants for these since these thresholds are used
+        # ubiquitously
+        FPL_200 = (
+            self.df[
+                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.LOW_INCOME_THRESHOLD
+        )
+
+        did_we_meet_threshold = (
+            self.df[
+                field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD_NAME
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        ) & FPL_200
+
+        self.df[field_names.FEMA_LOSS_RATE_LOW_INCOME] = did_we_meet_threshold
```

Comment:
nit: `did_we_meet_threshold` is a fairly vague name since it doesn't reference population loss. 

I'm wondering, can you just set: 

```
self.df[field_names.FEMA_LOSS_RATE_LOW_INCOME] = (
            self.df[
                field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD_NAME
                + field_names.PERCENTILE_FIELD_SUFFIX
            ]
            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
        ) & FPL_200
```

and skip the assignment?

Reply:
I'll look at this more closely

Reply:
Upon, review, that's certainly an idea I can roll with. It's just that in the event the interpreter hits a compilation error I know it's the variable instantiated rather than the assignment in which then I have to consider both the left and right - hand side.